README.md

```diff
@@ -1,6 +1,6 @@
 # docker-ansible
 
-For running Ansible.
+A convenient method for testing Ansible configuration management inside a Docker container. The docker-ansible image will load all defined environment dependencies into the container before executing the tests defined by your repo.
```

Comment:
I'd be down to rephrase the original line as "_Docker images for running Ansible playbooks._ " and put a rephrased form of your second sentence as point 3 in your Quick Start section:

> Dependencies are automatically sourced on-the-fly from `requirements.yml` for Ansible Galaxy, `requirements.txt` for PyPI, and `system_packages.txt` for yum or apk. Supports Ansible 2.3 and 2.4 on Alpine and CentOS 7.

My reasoning is mostly technical: this is an image, not a method, and it's not only for testing. On top of that, I feel like descriptions about behaviour are better left for usage instructions.